packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -586,6 +607,7 @@ const Autocomplete = <
       renderInput={renderInput}
       isOptionEqualToValue={getIsOptionEqualToValue}
       translate={translate}
+      filterOptions={filterOptions}
```

Comment:
Can we alphabetize this prop in the list? And, in order to avoid the problem I mentioned above, this should probably be 
```suggestion
      filterOptions={getOptions || filterOptions}
```

Reply:
Or, we just do `getOptions || null` and pass the `state` to the callback